### PR TITLE
Addressed comments from internal pilot #2 on setup

### PR DESCRIPTION
**Pull request by anenadic**
Addressed comments by @unode on 'Software Setup' section:
[Renato] What makes the material specific to Anaconda? Some packages? Is it specific to Anaconda (the distribution) or simply “conda” (the package manager)?
[Renato] Too used to “License” my typo sensor fired on “Licence” :) Today I learned! However in other parts of the lesson “License” is used (e.g. Introduction of episode 2 - “choosing a license”). Consider using US or UK spelling consistently.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -91,11 +91,13 @@ GitHub is a free, online host for Git repositories that you will use during the
 You can create an account at [https://github.com/](https://github.com/) if you don't already have one.
 
 ## Anaconda Python Distribution
-We will be using the Anaconda Python Distribution for the workshop. The material is not tested with other Python distributions and is specific to Anaconda.
+The material has been developed using the [Anaconda Python distribution](https://www.anaconda.com/) and is using the `conda` package manager 
+that comes with Anaconda. The material was not tested with other Python distributions and package managers but could
```
Comment (jag1g13) [inline setup.md:95]: How about, "The material has not been extensively tested with other Python distributions and package managers, but most sections are expected to work with some modification. For example, package installation and virtual environments would need to be managed differently."

Comment (jag1g13) [reply setup.md:95]: I agree we should push people to use conda, but if they're already familiar with `pip`, `venv`, etc. and know how to translate the notes for them, then they shouldn't be too disruptive.

The usage of conda in the CI section is probably the most difficult to replace, but it should work if they used system Python locally and conda for CI.

Comment (anenadic) [reply setup.md]: Fixed via e67a345.

Comment (anenadic) [reply setup.md]: Merging this branch.

**Pull request merged**